docs/document/website/versioned_docs/version-1.6.0/components/mobile/mobile-datetime-picker.md

```diff
@@ -0,0 +1,89 @@
+---
+id: version-1.6.0-mobile-datetime-picker
+title: MobileDateTimePicker
+sidebar_label: MobileDateTimePicker
+original_id: mobile-datetime-picker
+---
+
```

Comment:
Please create the same file with Japanese language at docs/document/website/translated_docs/ja/version-1.6.0